README.md

```diff
@@ -183,9 +183,11 @@ At the time of this writing, Fargate does not support `DAEMON` tasks (see [this]
 Furthermore, Fargate does not allow using `"pidMode": "host"` in the task definition (see documentation of `pidMode` [here](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_TaskDefinition.html)). Host PID is required for gProfiler to be able to profile processes running in other containers (in case of Fargate, other containers under the same `containerDefinition`).
 
 So in order to deploy gProfiler, we need to modify a container definition to include running gProfiler alongside the actual application. This can be done with the following steps:
-1. Modify the `command` parameter of your entry in the `containerDefinitions` array. The new command should include downloading of gProfiler & executing it in the background.
+1. Modify the `command` & `entrypoint` parameters of your entry in the `containerDefinitions` array. The new command should include downloading of gProfiler & executing it in the background, and `entrypoint` will be `bash`.
```

Comment:
I think we should specify that the `entrypoint` should be an array as well, like `"entryPoint": ["/bin/bash"]`. Otherwise the ECS JSON validation will pass, but the task creation will raise an error without any exact description what the actual issue is.